sale_cutoff_time_delivery/models/stock_picking.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from dateutil.relativedelta import relativedelta
+from psycopg2 import sql
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    cutoff_time_hms = fields.Char(
+        compute="_compute_cutoff_time_HMS", store=True, default="00:00"
```

Comment:
Could avoid this uppercase suffix?
```suggestion
        compute="_compute_cutoff_time", store=True, default="00:00"
```

Reply:
Renamed it but kept the `hms` in lower case.